src/styles/Book.css

```diff
@@ -0,0 +1,172 @@
+button {
+  margin: 0;
+  padding: 0;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+.bookItem {
+  border-radius: 4px;
+  background-color: #fff;
+  border: solid 1px #e8e8e8;
+  margin: 2.313rem 0 0 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  display: flex;
+  width: 85%;
+}
+
+.category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'RoboroSlab', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
```

Comment:
Great job @tomasesquivelgc  🤝

- [ ] As stated in the project requirements, kindly ensure there is no repeating of the same rule. Instead, create a single class to represent this rule and reuse this class where necessary. For example make sure the following classes are not repeated. `font-family: Montserrat, sans-serif;`, `font-family: 'RoboroSlab', sans-serif;`, etc. You may implement this across all your stylesheets. You may use this [article](https://speckyboy.com/good-bad-css-practices/) for more clarity.